seatunnel-connectors/seatunnel-connectors-flink/seatunnel-connector-flink-clickhouse/src/main/java/org/apache/seatunnel/flink/clickhouse/sink/client/ClickhouseClient.java

```diff
@@ -212,4 +221,12 @@ public ClickhouseTable getClickhouseTable(String database, String table) {
 
     }
 
+    public String localizationEngine(String engine, String ddl) {
```

Comment:
NIT: It's better to add UT for this method, and add comment what this method does.

Reply:
ok, i will add it later

Reply:
Please resolve the code style.